spec/features/user_show_spec.rb

```diff
@@ -49,13 +49,14 @@
     it 'I can see a button that lets me view all of a users posts.' do
       expect(page).to have_content 'See all posts'
     end
-    it 'When I click a users post, it redirects me to that posts show page.' do
-      visit "/user/#{@user2.id}/post/#{@post1.id}"
-      expect(page).to have_content 'post content 1'
+    it 'should redirect to that post show page when I click on a post' do
+      click_link 'post 2 title'
+      expect(page).to have_current_path("/user/#{@user2.id}/post/#{@post2.id}")
+      puts page.html
     end
     it 'When I click to see all posts, it redirects me to the users posts index page.' do
       click_link 'See all posts'
-      expect(page).to have_content 'post 1 title'
+      expect(page).to have_current_path("/user/#{@user2.id}/post")
     end

```

Comment:
- [Optional] Dear @Mike47ip, Good job making the suggested changes, But I noticed some other of your tests are still falling. When we write tests we need to make sure all the tests are passing. Please work on these tests and make sure all are passing. 

### The issue 
<img width="1136" alt="Screenshot 2023-11-18 at 6 48 45 in the evening" src="https://github.com/Mike47ip/Blog-app/assets/69077061/bb679be1-02ac-406f-be56-89a0a596e18d">